index.php

```diff
@@ -212,22 +212,16 @@
                                 </blockquote>
                             <?php break;
                             case "en": ?>
-                                <p>The artistic, historical and cultural richness is a very precious asset for all of our
-                                    country, Italy.
-                                    According to this parameter, Italy is the first country in the world. </p>
-                                <p>We believe that knowing the history and the particularities of the numerous places around us
-                                    is a way
-                                    to appreciate more what surrounds us.
-                                    It can also be a fun way to travel, get excited and discover places that are not very well
-                                    known, but
-                                    no less interesting.</p>
-                                <p>With Nice Places, for example, you can walk in the historic center of a city, you can consult
-                                    the app
-                                    to find out which places to visit, choose one and activate the navigator to reach it and
-                                    when you are
-                                    near a point of interest, you can open the description on the mobile phone to get a brief
-                                    overview of
-                                    what you are observing.</p>
+                                <p>The artistic, historical and cultural richness is a very precious asset for our country, Italy 
+                                    as well as many others. Italy boasts one of the oldest cities in the world, San Marino.  </p>
```

Comment:
I would like to keep the sentence `According to this parameter, Italy is the first country in the world.` instead of `Italy boasts one of the oldest cities in the world, San Marino.`
The other changes looks very good!